app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java

```diff
@@ -12,8 +13,6 @@
 import amazingme.app.EnumeratedActivity;
 import amazingme.model.AmazingMeAppCompatActivity;
 
-import static com.amazingme.activities.R.id.user_profile_back_button;
-
 public class ChildRegistrationActivity extends AmazingMeAppCompatActivity {
 
     //private Button backBtn, doneBtn;
```

Comment:
is there a reason you commented most of this file out?

Reply:
I wasn't sure whether the code would be needed for backend or not so I chose to comment it rather than erase